src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/passbook
 spring.datasource.username=postgres
-spring.datasource.password=qwerty1201
-#spring.jpa.hibernate.ddl-auto=create-drop
+spring.datasource.password=root
```

Comment:
this file should not change again and again else don't commit this file.

Reply:
Yes sir

Reply:
I can't see it's reverted